### PR TITLE
[Home] Added alternate banner

### LINKED ISSUE #1542: [Home] Hero: Create alternative hero: 5 icon
**Current Behavior**
The home page has a single hero created. 

**Desired Behavior**
Ideally, a few versions of the hero section are available to rotate through. There is a hero design with a large "5" icon in the design mockups that would be ideal to have.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by EmperorYP7**
Signed-off-by: Yash Pandey (YP) <yash.btech.cs19@iiitranchi.ac.in>

**Description**

This PR fixes #1542 

**Notes for Reviewers**

Screenshot 2021-02-27 at 10 11 24 PM

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@EmperorYP7 when building this, I see an empty hero section. The 5 doesn't show.

**Comment by EmperorYP7** [comment]
@leecalcote Fixed it!

**Comment by leecalcote** [comment]
Good, good. Before merging this one, let's fit it into the new banner rotation system that @vineethvanga18 created.

**Comment by EmperorYP7** [comment]
Sure! I'll refactor this as soon as #1640 is merged.

**Review by leecalcote — APPROVED** [review]
Screen Shot 2021-03-01 at 2 18 23 PM

unwanted margin at 900px width browser

**Comment by leecalcote** [comment]
https://user-images.githubusercontent.com/7570704/109554022-56379400-7a99-11eb-8eb3-59d2965d3f07.mov

@vineethvanga18 after the first rotation to the alt banner, we don't seem to be rotating back to the first banner. For this linear rotation, we'll want it to be circular, too.

File: src/sections/Home/Banner/index.js [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
     } else {
       sessionStorage.setItem("banner", 0);
     }
-  });
-  useEffect(() => {
     if (Banner != sessionStorage.getItem("banner")) {
       setBanner(sessionStorage.getItem("banner"));
     }
-  });
+  }, []);
```
Comment (EmperorYP7) [inline src/sections/Home/Banner/index.js:21]: Here, when we go back to home page for the 3rd time, it won't render `Banner-default` but `Banner-alt` which is not desired. Looking into it! 🔍

**Comment by vineethvanga18** [comment]
@leecalcote I made some updates just after you merged my PR. Please have a look at PR #1647

File: src/sections/Home/Banner/index.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const RotationalBanner = () => {
   const [Banner, setBanner] = useState(initialValue);
   useEffect(() => {
     if (sessionStorage.getItem("banner")) {
-      sessionStorage.setItem("banner", (1 + Number(Banner)) % BannersList.length);
+      sessionStorage.setItem("banner", (1 ^ Number(Banner)));
```
Comment (vineethvanga18) [inline src/sections/Home/Banner/index.js:20]: This only works for two banners

**Comment by l5io** [comment]
🚀 Preview for commit d5d65a0fabf03c4d7b0d4d751ffea12060b09bc4 at: https://60410dd4bcac774bce0a5674--layer5ng.netlify.app

File: src/sections/Home/Banner/index.js (hunk 2 above)
Comment (EmperorYP7) [reply src/sections/Home/Banner/index.js]: Oops. That was from my previous commits. Fixed! 👍🏻

**Comment by l5io** [comment]
🚀 Preview for commit dd09abca8842ebfd63130970f71a3a23fd50a679 at: https://60411267e4349200cf0b1fe8--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Gentlemen, after careful review, I have concluded that this is a very exciting piece of work. The rotating banner works well and the new alt-banner is stellar.

What's our 3rd banner? Oh, yes. Here it is - https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=1869%3A4138

**Pull request merged**